src/component/ConsentManagementPlatform.tsx

```diff
@@ -141,6 +144,14 @@ class ConsentManagementPlatform extends Component<Props, State> {
             active: false,
             visible: false,
         };
+
+        if (props.source) {
+            setSource(props.source);
+        }
+
+        if (props.variant) {
+            setSource(props.variant);
```

Comment:
I think you should be importing and calling `setVariant` here?

Reply:
👍